README.md

```diff
@@ -1,194 +1,122 @@
-# Superface One SDK _(one-sdk-js)_
-![GitHub Workflow Status](https://img.shields.io/github/workflow/status/superfaceai/one-sdk-js/CI)
-![NPM](https://img.shields.io/npm/v/@superfaceai/one-sdk)
-[![NPM](https://img.shields.io/npm/l/@superfaceai/one-sdk)](LICENSE)
-![TypeScript](https://img.shields.io/badge/%3C%2F%3E-Typescript-blue)
+[Website](https://superface.ai) | [Get Started](https://superface.ai/docs/getting-started) | [Documentation](https://superface.ai/docs) | [Discord](https://sfc.is/discord) | [Twitter](https://twitter.com/superfaceai) | [Support](https://superface.ai/support)
 
-<img src="https://github.com/superfaceai/one-sdk-js/blob/main/docs/LogoGreen.png" alt="superface logo" width="150" height="150">
+<img src="https://github.com/superfaceai/one-sdk-js/raw/main/docs/LogoGreen.png" alt="Superface" width="100" height="100">
 
-Superface is the core SDK of the Superface project. It is the library that communicates with registry and performs operations on profiles/maps, including input/output validations.
+# Superface OneSDK
 
-<!--TODO: Fill out this long description. So, should we fill it :) -->
+**Just one SDK for all the APIs you want to integrate with.**
 
-## Table of Contents
+[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/superfaceai/one-sdk-js/CI)](https://github.com/superfaceai/one-sdk-js/actions/workflows/main.yml)
+[![npm](https://img.shields.io/npm/v/@superfaceai/one-sdk)](https://www.npmjs.com/package/@superfaceai/one-sdk)
+[![license](https://img.shields.io/npm/l/@superfaceai/one-sdk)](LICENSE)
+![TypeScript](https://img.shields.io/static/v1?message=TypeScript&&logoColor=ffffff&color=007acc&labelColor=5c5c5c&label=built%20with)
+[![Discord](https://img.shields.io/discord/819563244418105354?logo=discord&logoColor=fff)](https://sfc.is/discord)
 
-- [Background](#background)
-- [Install](#install)
-- [Usage](#usage)
-- [Configuration](#configuration)
-- [Metric reporting](#metric-reporting)
-- [Security](#security)
-- [Support](#support)
-- [Maintainers](#maintainers)
-- [Contributing](#contributing)
-- [Licensing](#licensing)
-- [License](#license)
+OneSDK is a universal API client and core library of Superface. It provides a simple and uniform interface across many APIs and automatic runtime switching between API providers.
```

Comment:
drop "and core library of Superface"

Reply:
drop "a simple"

Reply:
"It provides unparalleled developer experience for every HTTP and Graph API" 
"The OneSDK brings unparalleled resiliency to API changes and comes with built-in integration monitoring, testing and fail-overs".

Reply:
Adapted like this:

> OneSDK is a universal API client which provides an unparalleled developer experience for every HTTP and Graph API. It enhances resiliency to API changes and comes with built-in integration monitoring, testing and fail-overs.